skore/src/skore/sklearn/_comparison/metrics_accessor.py

```diff
@@ -108,8 +109,11 @@ def report_metrics(
         scoring_kwargs : dict, default=None
             The keyword arguments to pass to the scoring functions.
 
-        pos_label : int, float, bool or str, default=None
-            The positive class.
+        pos_label : int, float, bool, str or None default=_DEFAULT
```

Comment:
```suggestion
        pos_label : int, float, bool, str or None, default=_DEFAULT
```
forgot to mention the other instances of this